build.gradle.kts

```
@@ -21,34 +23,20 @@ sourceSets["test"].java.srcDir("src/main/test")
 
 dependencies {
     implementation(kotlin("stdlib"))
-    implementation("junit:junit:4.12")
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3")
-    implementation("io.ktor:ktor-server-netty:$ktorVersion")
-    implementation("io.ktor:ktor-client-apache:$ktorVersion")
-    implementation("io.ktor:ktor-client-jackson:$ktorVersion")
-    implementation("io.ktor:ktor-locations:$ktorVersion")
-    implementation("io.ktor:ktor-jackson:$ktorVersion")
-    implementation("com.h2database:h2:1.4.200")
-    implementation("com.zaxxer:HikariCP:3.2.0")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.9.7")
-}
-
-application {
-    mainClassName = "no.bekk.kotlinworkshop.Task1Kt"
+    implementation("junit:junit:4.13.2")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.3")
+    implementation("io.ktor:ktor-server-netty-jvm:2.3.6")
```

Bruke variabelen her?
```suggestion
    implementation("io.ktor:ktor-server-netty-jvm:$ktorVersion")
```